infra/resources/_modules/function_apps/locals.tf

```diff
@@ -1,50 +1,69 @@
+    PidIssuerHealthCheckEnabled   = true
 
-      RevocationQueueName = var.revocation_queue_name
+    RevocationQueueName = var.revocation_queue_name
 
-      WEBSITE_SWAP_WARMUP_PING_PATH     = "/api/v1/wallet/health"
-      WEBSITE_SWAP_WARMUP_PING_STATUSES = "200"
-      },
-      local.function_apps.common_app_settings,
+    WEBSITE_SWAP_WARMUP_PING_PATH     = "/api/v1/wallet/health"
+    WEBSITE_SWAP_WARMUP_PING_STATUSES = "200"
+    },
+    local.function_apps.common_app_settings,
+    {
+      for s in var.user_func.app_settings :
+      s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.key_vault_wallet_name};SecretName=${s.key_vault_secret_name})" : s.value
+    }
+  )
+
+  function_app_user_slot_disabled = [
+    "addWalletInstanceToValidationQueue",
+    "validateWalletInstance",
+    "generateEntityConfiguration"
+  ]
+
+  function_app_user = {
+    app_settings = merge(local.function_app_user_common_app_settings,
       {
-        for s in var.user_func.app_settings :
-        s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.key_vault_wallet_name};SecretName=${s.key_vault_secret_name})" : s.value
+        for to_disable in local.function_app_user_slot_disabled :
+        format("AzureWebJobs.%s.Disabled", to_disable) => 0
```

Comment:
It is unclear why `to_disable` here is being set to 0 and why for the slot it is set to 1